xml/obs_build_config.xml

```diff
@@ -233,6 +233,32 @@ Keep: debbuild</screen>
         </para>
        </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><parameter>obsgendiff:</parameter></term>
+       <listitem>
+        <para>
+          OBS can run a program to compare the result of the current build with
```

Comment:
I don't think it actually does any comparison, it only extracts the data for external tools to be able to do so